app/models/item_change_set.rb

```diff
@@ -10,11 +10,12 @@ class ItemChangeSet
     license
     source_id
     use_statement
+    barcode
   ].freeze
 
-  def initialize
-    @changes = {}
-    yield self
+  def initialize(changes = {})
+    @changes = changes
+    yield self if block_given?
   end
```

Comment:
It looks like this change may be motivated by ease of testing, which... it's a simple and small class, so NBD, but I wonder if it's worth supporting two interaction modes for our change sets. An alternative to adding new initialization behavior for easier testing may be a new change set matcher?

Pinging @jcoyne on this for his thoughts.

Reply:
It wasn't testing per se. Just a more compact syntax for initializing.